app/Controllers/ProxyController.php

```diff
@@ -11,7 +12,8 @@ class ProxyController {
     static public function stream() {
         self::checkUrl();
         $url = $_GET['url'];
-        $streamer = new \TikScraper\Stream();
+        $config['USER_AGENT'] = Misc::env("user_agent", "Mozilla/5.0 (Android 12; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0");
```

Comment:
In order to make the PR more complete, you should also include this in `App\Helpers\Wrappers::api`, so both the video proxy and the regular API scraper can have a custom user agent.

Also, for the fallback value, instead of using a hardcoded string, you should just use `UserAgents::DEFAULT` from `TikScraper\Constants\UserAgents`.

Reply:
Alright, makes sense. I have now pushed a change to it.